app-editors/ghostwriter/ghostwriter-1.7.4.ebuild

```diff
@@ -25,6 +25,7 @@ RDEPEND="
 DEPEND="${RDEPEND}
 	dev-qt/linguist-tools:5
 	dev-qt/qtconcurrent:5
+	dev-qt/qtsvg:5
```

Comment:
ghostwriter is installing an svg icon. If this is used at runtime, you will also need to add qtsvg to `RDEPEND` even if it is not linked to.

Reply:
thanks, fixed.

Reply:
thanks, but an `RDEPEND` change requires revbump.